x/concentrated-liquidity/pool.go

```diff
@@ -85,7 +95,12 @@ type SwapState struct {
 }
 
 // this only works on a single directional trade, will implement bi directional trade in next milestone
-func (k Keeper) CalcOutAmtGivenIn(ctx sdk.Context, tokenIn sdk.Coin, tokenOutDenom string, swapFee sdk.Dec, minPrice, maxPrice sdk.Dec, poolId uint64) (newTokenIn, tokenOut sdk.Coin, err error) {
+func (k Keeper) CalcOutAmtGivenIn(ctx sdk.Context,
+	tokenIn sdk.Coin,
+	tokenOutDenom string,
+	swapFee sdk.Dec,
+	minPrice, maxPrice sdk.Dec,
```

Comment:
we can get rid of max and min price now right?

Reply:
this is to be done in upcoming PR